api/models/site.js

```diff
@@ -34,6 +44,22 @@ const beforeValidate = (site) => {
   }
 };
 
+
+function isEmptyOrUrl(value, throws = true) {
+  const validUrlOptions = {
+    require_protocol: true,
+    protocols: ['http', 'https'],
```

Comment:
The front end validation is looking for either `http` or `https`. Note the `?` after the `s` in the regex `https?://.+`